web/js/map-OpenLayers.js

```diff
@@ -801,6 +801,9 @@ OpenLayers.Strategy.FixMyStreetFixed = OpenLayers.Class(OpenLayers.Strategy.Fixe
 // params to /ajax if the user has filtered the map.
 OpenLayers.Protocol.FixMyStreet = OpenLayers.Class(OpenLayers.Protocol.HTTP, {
     read: function(options) {
+        // Show the loading indicator over the map
+        $('#loading-indicator').removeClass('hidden');
+        $('#loading-indicator').attr('aria-hidden', false);
```

Comment:
Does adding this mean the `Check we haven't received the data after the map has been clicked.` section below can be removed? As it's not possible to click the map whilst this indicator is visible and thus cause the race condition it's acting against.